doc/SConscript

```diff
@@ -107,6 +107,8 @@ if localenv['doxygen_docs']:
             '#/build/docs/doxygen/html', exclude=['\\.map', '\\.md5'])
 
 if localenv['sphinx_docs']:
+    if env["python_module"] is None:
+        logger.warn("Not building Sphinx docs because no Python module is available.")
```

Comment:
The reason why I did it this way is because a few lines below, there's:
```python
    env.Depends(sphinxdocs, env['python_module'])
```
so there needs to be a default value for `env['python_module']` anyways. I changed this check to `python_package` but kept the default value for `python_module` in `SConstruct`.